src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -14,7 +14,7 @@
             + "Past appointments are archived and can be accessed from the archive folder.";
     public static final String MESSAGE_EXPIRED_DATE_TIME = "The appointment date and time should be an upcoming one.";
     public static final String MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX = "The appointment index provided is "
-            + "invalid";
+            + "invalid or missing";
```

Comment:
Hmmm, I don't think this should be edited.

I took a look at the `ChangeCommandParser` and the message for line 42 (throw ParseException) should be `MESSAGE_INVALID_COMMAND_FORMAT` instead of `MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX`